core/src/main/scala/org/broadinstitute/dsde/rawls/config/RbsConfig.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsde.rawls.config
+
+import com.typesafe.config.Config
+
+case class RbsConfig(
```

Comment:
In general in this PR I think I'd prefer `ResourceBuffer` or something like it over `Rbs`